src/front-door/azext_front_door/commands.py

```diff
@@ -125,17 +125,34 @@ def load_command_table(self, _):
     # endregion
 
     # region WafPolicy
-    with self.command_group('network waf-policy', waf_policy_sdk) as g:
+    with self.command_group('network front-door waf-policy', waf_policy_sdk) as g:
         g.custom_command('create', 'create_waf_policy')
         g.command('delete', 'delete')
         g.command('list', 'list')
         g.show_command('show')
         g.generic_update_command('update', custom_func_name='update_waf_policy')
-        g.custom_command('set-managed-ruleset', 'set_azure_managed_rule_set')
 
-    with self.command_group('network waf-policy custom-rule', waf_policy_sdk) as g:
-        g.custom_command('create', 'create_wp_custom_rule')
-        g.custom_command('delete', 'delete_wp_custom_rule')
-        g.custom_command('list', 'list_wp_custom_rules')
-        g.custom_command('show', 'show_wp_custom_rule')
+    with self.command_group('network front-door waf-policy managed-rules', waf_policy_sdk) as g:
+        g.custom_command('add', 'add_azure_managed_rule_set')
+        g.custom_command('remove', 'remove_azure_managed_rule_set')
+        g.custom_command('list', 'list_azure_managed_rule_set')
+
+    with self.command_group('network front-door waf-policy managed-rules override', waf_policy_sdk) as g:
+        g.custom_command('add', 'add_override_azure_managed_rule_set')
+        g.custom_command('remove', 'remove_override_azure_managed_rule_set')
+
+    with self.command_group('network front-door waf-policy managed-rule-definition', waf_policy_sdk) as g:
+        g.custom_command('list', 'list_managed_rules_definitions')
+
+    with self.command_group('network front-door waf-policy rule', waf_policy_sdk) as g:
```

Comment:
You should be able to just put the `supports_local_cache` and `model_path` kwargs in the command group definition to eliminate the duplication. 

```Python
 with self.command_group('network front-door waf-policy rule', waf_policy_sdk, supports_local_cache=True, model_path="...") as g:
```